pkgs/build-support/setup-hooks/make-wrapper.sh

```diff
@@ -19,98 +19,98 @@
 #                                   are read first and used as VALS
 # --suffix-contents
 makeWrapper() {
-    local original=$1
-    local wrapper=$2
+    local original="$1"
+    local wrapper="$2"
```

Comment:
FYI, someone pointed out to me once that these quotes:

```
local original="$1"
```

are unneeded. But I tend to add them anyway :-)